app/models/claim.rb

```diff
@@ -105,6 +105,21 @@ def data_for_calculation
     }
   end
 
+  def is_nsm?
+    claim_type == ClaimType::NON_STANDARD_MAGISTRATE.to_s
```

Comment:
isn't this type of object always an nsm claim_type?

Reply:
No it can be an NSM claim or a breach of injunction

Reply:
So, horrifyingly, there are 2 "types" of non-standard magistrates claim. "non_standard_magistrate" and "breach_of_injunction". (Ivan mentioned this morning, this tripped him up when the requirement was that only non-standard magistrate claims are eligible for the youth court fee. He thought that meant "as opposed to PA applications", when actually it was "as opposed to breach of injunction claims"). It sucks, it's horrible naming, but it's what we have.

Reply:
In theory, it's a _really_ simple rename if we want to do it. We'd have to backfill every single claim, but it's a simple case of replacement.

Worthwhile to spike?

Reply:
ok, cheers! i good to know - that does feel like it could have been a property of a NSM claim though, maybe even a boolean?

Reply:
Before my time, but I suppose the thinking was other types of claims may be introduced down the line.